akka-http-core/src/main/scala/akka/http/model/HttpEntity.scala

```diff
@@ -38,13 +38,13 @@ sealed trait HttpEntity extends japi.HttpEntity {
   /**
    * A stream of the data of this entity.
    */
-  def dataBytes: Source[ByteString]
+  def dataBytes: Source[ByteString, Unit]
 
   /**
    * Collects all possible parts and returns a potentially future Strict entity for easier processing.
    * The Future is failed with an TimeoutException if the stream isn't completed after the given timeout.
    */
-  def toStrict(timeout: FiniteDuration)(implicit fm: FlowMaterializer): Future[HttpEntity.Strict] = {
+  def toStrict(timeout: FiniteDuration)(implicit fm: ActorFlowMaterializer): Future[HttpEntity.Strict] = {
```

Comment:
changed to ActorFlowMaterializer due to my rename. I am not sure where is enough to have a FlowMaterializer, but we should prefer it whenever possible
.